/**
 * @class MessageFactory
 * @author Sergey A Mikhailik
 * @date 09/10/16
 * @brief Wrapper for incoming/outgoing application message
 */

#ifndef OUTMESSAGEWRAPPER_H
#define OUTMESSAGEWRAPPER_H

#include "OLI_MsgDefines.h"

#include "protobuf/oli_messageslib.pb.h"

#include <string>

namespace oli
{
namespace msg
{
   
    /// declaration moved to the OLI_MsgDefines.h

}
}

#endif // OUTMESSAGEWRAPPER_H
